invoice/src/test/java/org/killbill/billing/invoice/template/formatters/TestDefaultInvoiceFormatter.java

```diff
@@ -187,6 +186,127 @@ public void testFormattedAmount() throws Exception {
     }
 
     @Test(groups = "fast")
+    public void testFormattedAmountFranceAndJPY() throws Exception {
+
+        final FixedPriceInvoiceItem fixedItem = new FixedPriceInvoiceItem(UUID.randomUUID(), UUID.randomUUID(), null, null,
+                                                                             UUID.randomUUID().toString(), UUID.randomUUID().toString(),
+                                                                             new LocalDate(), new BigDecimal("1500.00"), Currency.JPY);
+        final Invoice invoiceEUR = new DefaultInvoice(UUID.randomUUID(), new LocalDate(), new LocalDate(), Currency.JPY);
+        invoiceEUR.addInvoiceItem(fixedItem);
+
+        checkOutput(invoiceEUR,
+                    "<tr>\n" +
+                    "    <td class=\"amount\"><strong>{{invoice.formattedChargedAmount}}</strong></td>\n" +
+                    "</tr>\n" +
+                    "<tr>\n" +
+                    "    <td class=\"amount\"><strong>{{invoice.formattedPaidAmount}}</strong></td>\n" +
+                    "</tr>\n" +
+                    "<tr>\n" +
+                    "    <td class=\"amount\"><strong>{{invoice.formattedBalance}}</strong></td>\n" +
+                    "</tr>",
+                    "<tr>\n" +
+                    "    <td class=\"amount\"><strong>1 500,00 ￥</strong></td>\n" +
```

Comment:
> Another issue is that [DefaultAmountFormatter] (https://github.com/killbill/killbill/blob/master/util/src/main/java/org/killbill/billing/util/DefaultAmountFormatter.java#L23) is always [rounding] (https://github.com/killbill/killbill/blob/master/invoice/src/main/java/org/killbill/billing/invoice/template/formatters/DefaultInvoiceFormatter.java#L201) with scale = 2.

Weird, I cannot think of the reason why we forced that rounding (introduced by https://github.com/killbill/killbill/commit/0cab890a68df0b19c01886f2885ca9695d70600a). Maybe before we didn't enforce a default locale (https://github.com/killbill/killbill/commit/abed7b57fbdb6c2eef4bc188d50820a325b51f51)?

I would remove the rounding and verify other tests still pass.